contributors/guide/README.md

```diff
@@ -103,8 +103,8 @@ Another good strategy is to find a documentation improvement, such as a missing/
 
 #### Issue Assignment in Github
 
-Often, new contributors ask to be assigned an issue they are willing to take on. Unfortunately, due to GitHub limitations we can only assign issues to [org members](#community) or repo collaborators. 
-Instead, please state in a comment that you intend to work on this issue and it will be assumed to be yours.
+When you are willing to take on an issue, you can assign it to yourself. Just reply with `/assign` or `/assign @yourself` on an issue, 
```

Comment:
I think `/assign` will only work if they have commented on the issue/PR before the `/assign` comment...

Reply:
No, actually you can assign to yourself. 
You can refer to the statement from [GitHub](https://help.github.com/en/articles/assigning-issues-and-pull-requests-to-other-github-users).

> You can assign up to 10 people to each issue or pull request, **including yourself**, anyone who has commented on the issue or pull request, anyone with write permissions to the repository, and organization members with read permissions to the repository.

Reply:
I believe you can still only assign yourself (via the `/assign` comment) if you are an org member. 

Can you please restore that note about commenting that you intend to work on an issue for non-org members?

Reply:
More precisely, anyone can assign an issue to himself, even though he is non-org members.

Take me for example, i'm non-org member and i have assigned some issues for me.
https://github.com/kubernetes/kubernetes/issues/79610 .

By the way, I don't think we should present many rule here. Be simple is fine.

Reply:
Hmmm, I'm pretty sure this wasn't always possible and I'm not sure if it's intended behavior.
@kubernetes/owners -- did something change on our side or GH to allow this?

Reply:
This is new feature of GitHub.

Reply:
Ahhhhh, sorry. I didn't read the PR description! :)